app/controllers/decidim/action_delegator/admin/invite_participants_controller.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ActionDelegator
+    module Admin
+      class InviteParticipantsController < ActionDelegator::Admin::ApplicationController
+        include NeedsPermission
+
+        helper_method :current_setting, :users_list_to_invite, :participant, :form
+
+        def invite_user
+          enforce_permission_to :invite, :participant, resource: current_setting
+
+          Decidim::InviteUser.call(form) do
+            on(:ok) do
+              notice = t("invite_user.success", scope: "decidim.action_delegator.admin.invite_participants")
+              redirect_to decidim_admin_action_delegator.setting_participants_path(current_setting), notice: notice
+            end
+          end
+        end
+
+        def invite_all_users
+          enforce_permission_to :invite, :participant, resource: current_setting
+
+          users_list_to_invite.each do |participant|
```

Comment:
Can we put all this logic inside another Job? `InvitePendingParticipantsJob` for instance. This way we can move the method `users_list_to_invide` there and everything will look more tidy.